content/_data/indexpage/carousel.yml

```
@@ -1,25 +1,24 @@
-# Hacktoberfest 2024
-- :href: https://hacktoberfest.com/
-  :title: Hacktoberfest 2024
-  :intro: Jenkins is participating in Hacktoberfest 2024. We are seeking contributors
-    and maintainers who want to join us to improve Jenkins in October!
+- :href: /blog/2024/10/04/content-security-policy-grant/
+  :title: Alpha Omega Foundation Content Security Policy Grant
+  :intro: Alpha Omega Foundation has provided a grant to Jenkins to improve the implementation of Content Security Policy.
+    Read more about what this means for the Jenkins project.
   :image:
-    :src: /images/hacktoberfest/hacktoberfest_2024_social_600x300.png
-    :height: 300px
+    :src: /images/post-images/elections/2024/10/04/content-security-policy-grant.png
```

Extra element in the path needs to be removed.
```suggestion
    :src: /images/post-images/2024/10/04/content-security-policy-grant.png
```